boards/boardsource/Lulu/main.py

```diff
@@ -0,0 +1,220 @@
+import supervisor
+
+from kb import KMKKeyboard
+
+from kmk.extensions.peg_oled_Display import (
+    Oled,
+    OledData,
+    OledDisplayMode,
+    OledReactionType,
+)
+from kmk.extensions.peg_rgb_matrix import Rgb_matrix
+from kmk.handlers.sequences import send_string
+from kmk.hid import HIDModes
+from kmk.keys import KC
+from kmk.modules.layers import Layers
+from kmk.modules.modtap import ModTap
+from kmk.modules.split import Split, SplitSide, SplitType
+
+keyboard = KMKKeyboard()
+modtap = ModTap()
+layers_ext = Layers()
+keyboard.modules.append(layers_ext)
+keyboard.modules.append(modtap)
+
+oled_ext = Oled(
+    OledData(
+        corner_one={0: OledReactionType.STATIC, 1: ['qwertyzzzz']},
+        corner_two={
+            0: OledReactionType.LAYER,
+            1: ['1', '2', '3', '4', '5', '6', '7', '8'],
+        },
+        corner_three={
+            0: OledReactionType.LAYER,
+            1: ['base', 'raise', 'lower', 'adjust', '5', '6', '7', '8'],
+        },
+        corner_four={
+            0: OledReactionType.LAYER,
+            1: ['qwertyzzz', 'nums', 'shifted', 'leds', '5', '6', '7', '8'],
+        },
+    ),
+    toDisplay=OledDisplayMode.TXT,
+    flip=False,
+)
+keyboard.extensions.append(oled_ext)
+rgb_ext = Rgb_matrix(
+    ledDisplay=[
```

Comment:
Can this have a comment or be in a named variable describing what it is? this currently gives little explanation of what's going on

Reply:
@daysgobye Gonna need some info. I just did some formatting.